platform-sdk/swirlds-common/src/test/java/com/swirlds/common/config/export/ConfigExportTest.java

```diff
@@ -23,6 +23,7 @@
 import com.swirlds.common.metrics.config.MetricsConfig;
 import com.swirlds.config.api.Configuration;
 import com.swirlds.config.api.ConfigurationBuilder;
+import com.swirlds.config.extensions.export.ConfigExport;
```

Comment:
This test file should be moved to the corresponding folder/package instead of adding an import here.

Reply:
This test depends on several `swirlds-common` classes and therefore the (test) module would depend on `swirlds-common` and that I would not like to do